wasmcloud_host/Dockerfile

```diff
@@ -19,6 +19,14 @@ WORKDIR /opt/app
 # This copies our app source code into the build container
 COPY ./host_core ./host_core
 COPY ./wasmcloud_host ./wasmcloud_host
+
+# Install necessary system dependencies
+RUN apt update && \
```

Comment:
This is installing `git` in the builder image, right, not the actual runtime image? we shouldn't need git in the final execution container, right?

Reply:
Yup exactly, we only install and run what's strictly necessary in the release image